ambry-store/src/test/java/com.github.ambry.store/CompactionManagerTest.java

```diff
@@ -198,13 +320,28 @@ private void verifyCompactionDetails(CompactionDetails expectedCompactionDetails
   private class MockBlobStore extends BlobStore {
     private long usedCapacity;
     private long capacityInBytes;
+    private final CountDownLatch compactCallsCountdown;
     List<String> validLogSegments = null;
 
+    private boolean resumeCompactionCalled = false;
+    boolean compactCalled = false;
+    Exception callOrderException = null;
+
+    RuntimeException exceptionToThrowOnResume = null;
+    RuntimeException exceptionToThrowOnCompact = null;
+    boolean started = true;
+
     MockBlobStore(StoreConfig config, StorageManagerMetrics metrics, Time time, long capacityInBytes,
         long usedCapacity) {
+      this(config, metrics, time, capacityInBytes, usedCapacity, new CountDownLatch(0));
```

Comment:
minor: can you pass in storeId in constructor of MockBlobStore() and add the storeId in all log error statements. Would help us during debugging.